ci/ray_ci/automation/test_docker_tags_lib.py

```diff
@@ -429,5 +430,111 @@ def test_backup_release_tags(
         assert call_arg.kwargs["tag"] == f"{namespace}/{repository}:2.0.{i}"
 
 
+@pytest.mark.parametrize(
```

Comment:
send in the lists as args so that the result is not hard coded to the real list and can be maybe shorter too?

we will need to update the list when changing versions, and making changes should not require changing the tests here if the logic is not changed.

Reply:
I updated the unit test to be shorter with the list passed in as args